android/app/src/main/java/org/fossasia/openevent/utils/NotificationUtil.java

```diff
@@ -35,7 +35,7 @@ public static Completable createNotification(Session session, Context context) {
             } else if (pref_result.equals(12)) {
                 zonedDateTime.minusHours(12);
```

Comment:
Shouldn't this also be changed @manav-mj ?

Reply:
<img src="https://user-images.githubusercontent.com/22222147/30658788-7cba2842-9e59-11e7-95c5-7b13629861db.png" height = "480" width="270">

We have the option for
1. 10 min
2. 1 hour
3. 12 hour